l1-contracts/contracts/bridge/interfaces/IL2WethBridge.sol

```diff
@@ -2,6 +2,11 @@
 
 pragma solidity 0.8.20;
 
-interface IL2WethBridge {
-    function initialize(address _l1Bridge, address _l1WethAddress, address _l2WethAddress) external;
+interface IL2SharedBridge {
```

Comment:
You should rename the contract file too as it still `IL2WethBridge.sol`

Reply:
We never actually use this, will delete. But true for L2ERC20Bridge

Reply:
Actually we never use either